tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaApisTest.java

```diff
@@ -314,6 +316,65 @@ public void testConsumerListOffsetLatest() throws Exception {
         assertEquals(listOffsetResponse.responseData().get(tp).timestamp, Long.valueOf(0));
     }
 
+    /**
+     * Test the sending speed of fetch request when the readable data is less than fetch.minBytes.
+     */
+    @Test(timeOut = 60000)
+    public void testFetchMinBytes() throws Exception {
+        String topicName = "testMinBytesTopic";
+        TopicPartition tp = new TopicPartition(topicName, 0);
+
+        // create partitioned topic.
+        admin.topics().createPartitionedTopic(topicName, 1);
+        List<TopicPartition> topicPartitions = new ArrayList<>();
+        topicPartitions.add(tp);
+
+        int maxResponseBytes = 800;
+        int maxPartitionBytes = 900;
+        int maxWaitMs = 10000;
+        //case1: consuming an empty topic
+        KafkaHeaderAndRequest fetchRequest1 = createFetchRequest(maxWaitMs,
+                1,
+                maxResponseBytes,
+                maxPartitionBytes,
+                topicPartitions,
+                Collections.EMPTY_MAP);
+        CompletableFuture<AbstractResponse> responseFuture1 = new CompletableFuture<>();
+        Long startTime1 = System.currentTimeMillis();
+        kafkaRequestHandler.handleFetchRequest(fetchRequest1, responseFuture1);
+        responseFuture1.get();
+        Long endTime1 = System.currentTimeMillis();
+        System.out.println("cost time1:" + (endTime1 - startTime1));
+
+        //case2: consuming an  topic after produceing data
+        KafkaProducer<String, String> kProducer = createKafkaProducer();
+        produceData(kProducer, topicPartitions, 10);
+
+        Map<TopicPartition, Long> offsetMaps = Maps.newHashMap();
+        offsetMaps.put(tp, 0L);
+
+        Map<TopicPartition, Long> offsetMap = new HashMap<>();
```

Comment:
```suggestion
```

Remove unused variable to make Codacy check pass. We use Codacy check as a way to guarantee code quality, please notice the check result when you push commits.